crates/ruff_python_whitespace/src/tokenizer.rs

```diff
@@ -18,30 +18,22 @@ pub fn first_non_trivia_token(offset: TextSize, code: &str) -> Option<Token> {
         .next()
 }
 
-/// Returns the first non-trivia token right before `offset` or `None` if at the start of the file
-/// or all preceding tokens are trivia tokens.
-///
-/// ## Notes
-///
-/// Prefer [`first_non_trivia_token`] whenever possible because reverse lookup is expensive because of comments.
-pub fn first_non_trivia_token_rev(offset: TextSize, code: &str) -> Option<Token> {
```

Comment:
I removed this method because I intentionally want to make it hard to do backward lexing.